listener/types.go

```diff
@@ -216,6 +217,12 @@ func NewEthListenJob(jobType int, listener bridgeCore.Listener, subscriptionName
 	}
 }
 
+func (e *EthListenJob) Process() ([]byte, error) {
```

Comment:
In bridge core, the Process of Listen job store job to db, do we need to do it here?
```go
func (e *BaseJob) Process() ([]byte, error) {
	// save event data to database
	if e.jobType == ListenHandler {
		subscription, ok := e.listener.GetSubscriptions()[e.subscriptionName]
		if ok {
			if err := e.listener.GetStore().GetEventStore().Save(&models.Event{
				EventName:       subscription.Handler.Name,
				TransactionHash: e.tx.GetHash().Hex(),
				FromChainId:     hexutil.EncodeBig(e.FromChainID()),
				CreatedAt:       time.Now().Unix(),
			}); err != nil {
				log.Error(fmt.Sprintf("[%sListenJob][Process] error while storing event to database", e.listener.GetName()), "err", err)
			}
		}
		return e.data, nil
	}
	return nil, nil
}
```

Reply:
I called this at parent function, so basically we don't need to add the stats to here